vtr_flow/tasks/regression_tests/yosys+odin/vtr/config/config.txt

```diff
@@ -0,0 +1,58 @@
+##############################################
```

Comment:
@vaughnbetz would you mind letting me know where is the appropriate place for the Yosys+Odin-II task. It runs the VTR flow over the VTR benchmarks, and with eight cores, it roughly takes 10-12 hours.

Reply:
That's quite a while!  I think we'll need a new vtr_reg_nightly to capture that (vtr_reg_nightly5) and may have to cut it into two pieces (vtr_reg_nightly5 and 6) to keep its runtime from limiting our turnaround.

Adding @ArashAhmadian to comment on what the longest running tests currently are (I think they are 6 hours wall clock, so we should make everything les than that).

@sdamghan : an this test be pared back at all?

Reply:
@vaughnbetz not sure what would you mean by "test be pared back," would you please rephrase it?

Reply:
There are a few ways to cut cpu time:
- Remove a few of the largest benchmarks
- Break this into two tests and only run the largest benchmarks on one architecture
- Break into two tests and only run at a fixed channel width (--route_chan_width 300 , for example) for the test that includes the largest / slowest to compile circuits.

Also, what makes this script run yosys + odinII?  I don't see any script parameters that point that out.

Reply:
I have not set any script parameter to run this task yet. Removing the `k6_frac_N10_40nm` architecture would considerably decrease the run-time since the synthesis is not elaborated in pure soft logic. By running only through the second architecture, I guess it would take less than 1 or 2 hours by running only through the second architecture. Once I set them up, I will notify you. Will also compare the runtime for fixed and variable channel width

Reply:
The longest running suite is vtr_reg_nightly_test2 (~5-6 hrs wall-clock, bottlenecked by titan_quick_qor). You should also make sure that the largest circuits are put at the bottom of circuit_list so they all start at the beginning of the run and don't stack on top of each other. This also may help shave off some time if you haven't already done it.

Reply:
Thanks, @ArashAhmadian , Once I get it done, I will notify you to have a look to make sure about their consistency.

Reply:
Sounds good.